File: Anyelo Carrillo Java/Tabla de la verdad.java
public class Tabla_de_verdad {
    public static void main(String[] args) {
        // Parte 1: Operaciones con booleanos
        boolean a = true;
        boolean b = false;
        System.out.println(a && b); // Imprime "false"

        // Parte 2: Operaciones con enteros y comparaciones
        int aNum = 2;
        int bNum = 3;
        int c = 4;
        int d = 10;

        // Comparación 1: a == b y c < d
        System.out.println((aNum == bNum) && (c < d)); // Imprime "false"

        // Comparación 2: not (a == b) y c > d
        System.out.println(!(aNum == bNum) && (c > d)); // Imprime "false"
    }
}

File: Anyelo Carrillo Go/Tabla de la verdad.go
package main

import "fmt"

func main() {
	// Operaciones con booleanos
	a := true
	b := false
	fmt.Println(a && b) // Imprime "false" (equivalente a "and" en Python)

	// Operaciones con enteros y comparaciones
	aInt := 2
	bInt := 3
	c := 4
	d := 10

	// Comparación 1: a == b and c < d
	fmt.Println(aInt == bInt && c < d) // Imprime "false"

	// Comparación 2: not a == b and c > d
	fmt.Println(!(aInt == bInt) && c > d) // Imprime "false"
}
